compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -17,6 +17,7 @@
 */
 package org.wso2.ballerinalang.compiler.semantics.analyzer;
 
+import io.ballerina.runtime.api.utils.IdentifierUtils;
```

Comment:
I'm not sure if we should be importing something from the runtime API here. 

@warunalakshitha, @HindujaB, thoughts and/or any alternatives?

Reply:
I think we use this in so many places, agreed that it's better to be in a commonplace.

Reply:
Yes. We already discussed moving to the new common module. Please create an issue for this.

Reply:
Created issue #33769